system/API/ResponseTrait.php

```diff
@@ -53,39 +52,40 @@
  */
 trait ResponseTrait
 {
+
 	/**
 	 * Allows child classes to override the
 	 * status code that is used in their API.
 	 *
 	 * @var array
 	 */
 	protected $codes = [
-		'created'                   => 201,
-		'deleted'                   => 200,
-		'invalid_request'           => 400,
+		'created' => 201,
```

Comment:
Personally, the more I work with different systems the more I believe the only way to keep consistency is to go the PSR2 route and have everything be spaces, but that's just me....